src/index.test.ts

```diff
@@ -3,11 +3,11 @@ import * as path from 'path';
 
```

Comment:
<!--
  3 failure:  " should be ' lin..., tab indentation e..., unused expression...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: src/index.test.ts;
  Line: 3;
-->
- :no_entry_sign: " should be ' line - .travis.yml (quotemark)
- :no_entry_sign: tab indentation expected line - .travis.yml (indent)
- :no_entry_sign: unused expression, expected an assignment or function call line - .travis.yml (no-unused-expression)